### ISSUE TITLE
Version for laravel 4.2

### ISSUE THREAD

**Issue by cdarken** [crhayes/laravel-extended-validator#15]
It would have been nice to have a final release for laravel 4.2, as I was using it in an application and the recent update that requires 5.0 broke my deps.

**Comment by ThomasSimoens** [comment]
Yeah, the 0.7 version (compatible with laravel 4.2) has a number of missing breaking changes, including the (new) protected visibility of `getRulesInContext()`, `bindReplacements()` and `hasContext()` in ContextualValidor

Thanks @cdarken
